docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -858,6 +860,21 @@ By default, the users' organization and role is reset on every new login.
 > With Grafana 10, if `oauth_skip_org_role_update_sync` option is set to `false`, users with no mapping will be
 > reset to the default organization role on every login. [See `auto_assign_org_role` option]({{< relref ".#auto_assign_org_role" >}}).
 
+### [auth].<provider> skip_org_role_update_sync
```

Comment:
@IevaVasiljeva updated the PR with some documentation as well for reflecting what happens specifically to the org roles with the different settings set.

Hope this makes sense for the user.

Also; the placement of this is meant for now to be alongside the "generic" setting to make it explicit that we are working towards making this the default.

Not sure where to place this otherwise 🤔 , any pointers there?

Reply:
I would probably prefer to have separate explicit sections for each provider (ie, this one would be [auth.grafana_com] skip_org_role_update_sync. That would also make it very clear for which providers we have a provider specific setting and for which we don't have it yet.

Reply:
let's do one by one first. Thought it might make it a lot of documentation and maybe each provider might be the same settings to reflect. Who knows?

Let's start with one by one!